src/addressUtilsShelley.c

```diff
@@ -503,3 +516,41 @@ void view_parseAddressParams(read_view_t* view, addressParams_t* params)
 		ASSERT(false);
 	}
 }
+
+static inline bool isSpendingPathConsistentWithAddressType(const address_type_t addressType, const bip44_path_t* spendingPath)
```

Comment:
it most likely can, the code is currently a bit of a hybrid between the operator app branch where a few refactors regarding recognition of paths have been done and develop - we already agreed with @janmazak that he'll extract the refactors from the operator app to be usable here, eliminating this duplicity

Reply:
not sure if still relevant as I did a few rebases since then - now I see this function being used only within addressUtilsShelley.c - do you still see a point in reusing this function outside this file?

Reply:
Nope, it seems that the function has been removed from `securityPolicy.c` and is now only in `addressUtilsShelley.c`. :+1: